patcher/src/widgets/CharacterCreation/services/session/attributeOffsets.ts

```diff
@@ -43,22 +43,22 @@ export function fetchAttributeOffsetsSuccess(offsets: AttributeOffsetInfo[]) {
   };
 }
 
-export function fetchAttributeOffsetsFailed(error: ResponseError) {
```

Comment:
Actually, I think what really needs to happen is GameDataAPI needs to add some proper typing.